src/SessionManager.ts

```diff
@@ -6,6 +6,10 @@ import { RequestClient } from './request/RequestClient'
 
 const MAX_SESSION_COUNT = 1
 
+interface ErrorResponse {
```

Comment:
we already have a class named ErrorResponse. It can be conflicting with that. Can you update this or modify that class to your requirements

Reply:
You have a point. I renamed it to `ApiErrorResponse`